### PR TITLE
Dev: Locking down application headers for security

### PR DESCRIPTION
**Pull request by kalpaitch**
@nao and I had a conversation earlier about iFraming content, which made me realise we need to lock down our security headers - https://securityheaders.io/

Enabling a very simple module (one wot I wrote earlier) to adapt some potentially leaky headers.

### CODE REVIEW

File: sync/http_response_headers.response_header.x_powered_by.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+uuid: b6c5660e-ab6f-4caf-9213-c10c770cdd32
+langcode: en
+status: true
+dependencies: {  }
+_core:
+  default_config_hash: _Bau76mGw4kCqnHi4BTCC-Uq2PS0Ih5hZ94lblLewEE
+id: x_powered_by
+label: X-Powered-By
+description: 'The X-Powered-By header gives information on the technology that''s supporting the Web Server. It is best not to provide this information.'
```
Comment (stemount) [inline sync/http_response_headers.response_header.x_powered_by.yml]: does this escape the `'`?

**Comment by Nao** [comment]
@kalpaitch, I don't think that's me you were referring to..?

**Comment by kalpaitch** [comment]
@Nao It was vaguely related to iFraming content for the marketing site. Not a biggie, but not an issue no more.

**Comment by Nao** [comment]
Yes, but I have no idea what 'TransformCore' is, or what 'iFraming' is, so I doubt it's me really.. ;)
I'm the author of Wedge, Kyodai and Lestrade's. If you don't know those either, chances are you poked the wrong person. ;)

**Pull request merged**